website/blog/2024-09-24-run-ferretdb-cybertec-postgres-kubernetes.md

```diff
@@ -0,0 +1,350 @@
+---
+slug: run-ferretdb-cybertec-postgres-kubernetes
+title: 'Run FerretDB on Kubernetes with CyberTec Postgres'
+authors: [alex]
+description: >
+  Learn how to run FerretDB on Kubernetes using CyberTec Postgres operator for a truly open source alternative to MongoDB.
+image: /img/blog/ferretdb-cybertec-postgres.jpg
+tags: [compatible applications, tutorial, cloud, postgresql tools, open source]
+---
+
+![Run FerretDB and Postgres Cluster using CyberTec Postgres on Kubernetes](/img/blog/ferretdb-cybertec-postgres.jpg)
+
+Are you looking to replace your MongoDB instance with an open source solution?
+You can do that by running FerretDB on Kubernetes with CyberTec Postgres.
+
+<!--truncate-->
+
+[FerretDB](https://www.ferretdb.com/) offers a truly open-source document database alternative to MongoDB, removing the risks associated with vendor lock-in and the cost implications of proprietary licenses.
+By using Postgres as the underlying database, you gain the reliability, scalability, and extensive feature set of one of the most reliable databases available today.
+With [CyberTec Postgres operator](https://www.cybertec-postgresql.com/en/), you can enable production-grade features such as auto-failover, rolling updates, and automated backups, suitable for enterprise-level workloads.
+
+In this blog post, you'll learn to set up FerretDB with CyberTec Postgres as the backend on Kubernetes.
+
+## Prerequisites
+
+- Kubernetes cluster (use [Minikube](https://minikube.sigs.k8s.io/docs/start/) for local development and testing)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Helm](https://helm.sh/docs/intro/install/)
+- [psql](https://www.postgresql.org/docs/current/app-psql.html)
+
+## Guide to setup CyberTech Postgres operator
+
+FerretDB is a truly open source MongoDB alternative database that uses Postgres as a backend.
+To run FerretDB on Kubernetes, you need to set up a Postgres cluster, and you can do that using the CyberTec Postgres operator.
+
+### Download the CyberTec project
+
+Start by downloading/cloning the the CyberTec project.
+
+```sh
+GITHUB_USER='USERNAME'
+git clone https://github.com/$GITHUB_USER/CYBERTEC-operator-tutorials.git
```

Comment:
Unless you've forked the repository, the link to clone won't exist. I suggest just cloning original one, I believe that's `github.com:cybertec-postgresql/CYBERTEC-operator-tutorials.git`. Please check it's the correct repo.